Client/Frontend/Library/BookmarkDetailPanel.swift

```diff
@@ -205,7 +210,11 @@ class BookmarkDetailPanel: SiteTableViewController {
             }
 
             if bookmarkNodeType == .bookmark {
-                return profile.places.updateBookmarkNode(guid: bookmarkNodeGUID, parentGUID: parentBookmarkFolder.guid, title: bookmarkItemOrFolderTitle, url: bookmarkItemURL)
+                guard let bookmarkItemURL = self.bookmarkItemURL, let fixedUpBookmarkItemURL = URIFixup.getURL(bookmarkItemURL) else {
```

Comment:
Same as above. If you're validating already to disable the save button when the URL field changes, you don't need to do anything here.

Reply:
Same response as above - this is needed to potentially change the URL.